crates/cli-support/src/lib.rs

```diff
@@ -327,6 +327,13 @@ impl Bindgen {
                 .context("failed getting Wasm module")?,
         };
 
+        // Check that no exported symbol is called "default" if we target web.
+        if matches!(self.mode, OutputMode::Web)
```

Comment:
I'm unfamiliar with the other targets, but does this really only affect the `web` target?

Reply:
As far as I can see, then yes. I tried to compile a dummy crate for all targets and grep'ing for default in each of the generated JS files. I only found default to be present in `web`.

Reply:
My question was if the Web target is the only target having issues with the identifier `default`.

If the other targets didn't contain `default`, it would mean they have an issue exporting it. Web shouldn't contain `default` because you made it error out.

So I'm a bit confused about whats going on.

Reply:
No you misunderstood me. What I did to sanity check this was to create a new Rust project that does not export any symbol called `default`, e.g.  lib.rs looks something like this:
```rust
use wasm_bindgen::prelude::*;

#[wasm_bindgen]
pub fn greet() {
    alert("Hello, wasm-target-test!");
}
```
then for each target `bundler, nodejs, web, no-modules` I checked that each generated JS file didn't contain or use the symbol `default`or the keyword `export default <foo>`. Which is true for every target except `web`